packages/vite/src/node/optimizer/index.ts

```diff
@@ -724,12 +724,25 @@ export function runOptimizeDeps(
         return successfulResult
       })
 
-      .catch((e) => {
+      .catch(async (e) => {
         if (e.errors && e.message.includes('The build was canceled')) {
           // esbuild logs an error when cancelling, but this is expected so
           // return an empty result instead
           return cancelledResult
         }
+        const prependMessage = colors.red(`\
+Error during dependency optimization:
+
+`)
```

Comment:
```suggestion
        const prependMessage = colors.red('Error during dependency optimization:\n\n')
```

I think this can be a bit shorter so it doesn't take up multiple lines, and as there's no interpolation.